src/Uno.UI/UI/Xaml/Controls/Frame/Frame.cs

```diff
@@ -192,6 +192,19 @@ private void OnSourcePageTypeChanged(DependencyPropertyChangedEventArgs e)
 
 		#endregion
 
+		#region IsNavigationStackEnabled DependencyProperty
+
+		public bool IsNavigationStackEnabled
+		{
+			get { return (bool)GetValue(IsNavigationStackEnabledProperty); }
+			set { SetValue(IsNavigationStackEnabledProperty, value); }
+		}
+
+		public static readonly DependencyProperty IsNavigationStackEnabledProperty =
```

Comment:
```suggestion
		public static readonly DependencyProperty IsNavigationStackEnabledProperty =>
```

Don't change the property to a field

Reply:
I know but all other DependencyProperty declarations are fields 🤔

Reply:
I could make this one a property and the we could add a task to convert every "field" DependencyProperty in the codebase to property to the breaking change rollup.

Reply:
Made #2953 for this